### ISSUE TITLE
confusion about applying slots from Readme.md

### ISSUE THREAD

**Issue by ulno** [Daenara/rhasspy_weather#6]
I am now trying to use rhasspy_weather in Rhasspy, but I have a little trouble understanding where the slots go?
If I understood the manual (https://rhasspy.readthedocs.io/en/latest/training/#slots-lists) correctly, slots are just files? So, where would your json-slots listing go then, or should this be input for a python parser slot program that returns the respective sub-lists?

**Comment by Daenara** [comment]
Basically the slots go into the slot tab in the webGUI. That is where I copied them from and that is then split up into single files in the slots folder (as fas as I know, I don't understand rhasspy that well, just know how to use it). The slots folder contains files without extension, named after the slot containing just a plain ini style list. I provided the whole json for the gui because that is easier than to explain ppl how to name each file and where to put it.

(though I have no idea if 2.5 still works the same)
